orbit/cmd/orbit/orbit.go

```diff
@@ -666,6 +666,8 @@ func main() {
 		defer cancel()
 		g.Add(signalHandler(ctx))
 
+		go sigusrListener(c.String("root-dir"))
+
```

Comment:
I suspected there would be an issue but I haven't tested it, Added and I'll se if that fixes the Windows issue

Reply:
sounds good! a heads-up: we need `sigusrListener` but seems like `dumpProf` is defined, (typo since both have the same signature?)

Reply:
I thought putting them into separate functions would make it cleaner since I want to keep the listener running even if there's an error while dumping the prof data. I guess I could put `sigusrListener` into a `go func(){` thing in orbit.go but that file already seemed too large for my liking

Reply:
yes and that looks great to me, I meant that we need to define `sigusrListener` in `orbit/cmd/orbit/signal_windows.go` as is the function that's actually called from `orbit.go`, otherwise you'll get the same error

Reply:
oh, uuuh, oops
Thanks!